app/assets/javascripts/discourse/app/templates/login.hbs

```diff
@@ -161,9 +161,9 @@
 
           <DButton
             @action={{if
-              this.siteSettings.experimental_full_page_login
-              this.showLoginPage
+              this.shouldShowLogin
               (route-action "showLogin")
+              this.showLoginPage
```

Comment:
Naming is a little tricky here... I am finding it hard to follow `shouldShowLogin` and `showLoginPage`. Is there any renaming we can do here to make this easier to follow?